fischkapp/src/components/cards/FrontCard.tsx

```diff
@@ -0,0 +1,22 @@
+import { CardsWrapper } from "../styles/SitesCards.styled";
+import { StyledInput } from "../styles/SitesCards.styled";
+import { StyledCardButton } from "../styles/buttons/CardButton";
+import { StyledButtonsContainer } from "../styles/SitesCards.styled";
+
+type FrontCardProps = {
+  onFlipCard: Function;
```

Comment:
**nitpick**: We should type functions explicitly - in this case it should be: onFlipCard: () => void